copycat/file/src/main/java/org/apache/kafka/copycat/file/FileStreamSourceTask.java

```diff
@@ -193,4 +190,12 @@ public void stop() {
             this.notify();
         }
     }
+
+    private Map<String, String> offsetKey(String filename) {
```

Comment:
Do you think we can turn these into more generic offset utilities that all connectors can share? 
I saw in your comments that you had some ideas.

Do you think it belongs in this patch? or a followup?